code-challenges.js

```diff
@@ -10,54 +10,23 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-//Make a function that is called temperatureChecker
-const temperatureChecker = (temp) => {
-  //Create an if statement stating that temp is below boiling point
-  if (temp < 212){
-    //return "35 is below boiling point"
-    return `${temp} is below boiling point`
-  }
-  //Create an else if statement stating that temp is above boiling point
-  else if (temp > 212) {
-    //return "350 is above boiling point"
-    return `${temp} is above boiling point`
-  }
-  //Create an else if statement stating that temp is at boiling point
-  else if (temp === 212){
-    //return "212 is at boiling point"
-    return `${temp} is at boiling point`
-    //Create else "broken"
-  } else {
-    return "broken"
-  }
-}
-//console.log(temperatureChecker(temp)) and put in variables given above
-console.log(temperatureChecker(35));
-console.log(temperatureChecker(350))
-console.log(temperatureChecker(212))
+
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var arr = [3, 7, 0, 6, -9]
-//Create for loop
-for(let i=0; i<arr.length; i++){
-  //console.log() and have the array being multiplied by 5
-  console.log(arr[i] * 5)
-}
+var myNumbers1 = [3, 7, 0, 6, -9]
+
+
+
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
-//create a const variable to make sure the variable name doesn't change
-//make it a function
-//use the map method to be able to go thorugh the array and multiply each index by 5
-const multiplyer = myNumbers2.map(value => value * 5)
```

Comment:
this one is also suuuuper close! this should just be wrapping a function. right now it is just a variable with a `const` assignment. like this:
```javascript
const multiplyer = (arr) => arr.map(value => value * 5)
```